### ISSUE TITLE
Unable to find a spec satisfying helm-rb on MacBook with Apple M1 chip

### ISSUE THREAD

**Issue by mattruzicka** [getkuby/helm-rb#1]
I'm receiving the below error when trying to install kuby-core on my laptop with the Apple M1 chip

```
Unable to find a spec satisfying helm-rb (~> 0.1) in the set. Perhaps the lockfile is
corrupted? Found helm-rb (0.1.0-x86_64-linux), helm-rb (0.1.0-x86_64-darwin), helm-rb
(0.1.0-s390x-linux), helm-rb (0.1.0-ppc64le-linux), helm-rb (0.1.0-i386-linux), helm-rb
(0.1.0-arm64-linux), helm-rb (0.1.0-arm-linux) that did not match the current platform.
```

**Comment by camertron** [comment]
Hey @mattruzicka, thanks for letting me know about this. You're right, both helm-rb and kubectl-rb don't have Darwin ARM versions yet. I just added support to both projects and published a bunch of new gems as v0.2.0. Try installing again and let me know if things are working :)

**Comment by mattruzicka** [comment]
I was able to install today, thanks!
